staging/src/k8s.io/apiserver/pkg/authentication/request/x509/x509.go

```diff
@@ -110,6 +111,15 @@ func certificateIdentifier(c *x509.Certificate) string {
 	)
 }
 
+func certificateErrorIdentifier(c *x509.Certificate) string {
```

Comment:
`GetHumanCertDetail` might be better, though it could make the audit log large?

Reply:
Perhaps `GetPrivateCertDetail` to underline that it contains sensitive data?

Reply:
`GetHumanCertDetail` is an existing function in `k8s.io/apiserver/pkg/server/dynamiccertificates`.

Reply:
Oh, I see, got it. Using it now - although I had to update it to truncate current time to seconds to pass unit tests (not before and not after are already being displayed like that now)